packages/custom-field/test/visual/lumo/custom-field.test.js

```diff
@@ -97,6 +97,12 @@ describe('custom-field', () => {
       element.setAttribute('theme', 'helper-above-field');
       await visualDiff(element, 'helper-above-field');
     });
+
+    it('theme-whitespace', async () => {
+      element.setAttribute('theme', 'whitespace');
+      element.label = 'Label';
+      await visualDiff(element, 'whitespace-theme');
```

Comment:
As discussed, let's adapt this test to use a block element as child, which makes it more obvious that whitespace adds an offset between label and child.